gio-macros/tests/actions.rs

```diff
@@ -0,0 +1,79 @@
+use gio::prelude::*;
+use std::cell::RefCell;
+use std::rc::Rc;
+
+#[derive(glib::Downgrade)]
+pub struct MyActionGroup {
```

Comment:
How does this interfer with an actual `GActionGroup` implementation? @bilelmoussaoui

Reply:
It doesn't. The macro just generates one method (`register_actions`) which allows you to register all actions in an arbitrary `ActionMap`.

`GSimpleActionGroup` is used here because it is the only class in `gio` which implements both `GActionMap` and `GActionGroup` and can be tested end-to-end.